src/Components/Settings.tsx

```diff
@@ -106,7 +110,9 @@ export const Settings: React.FC = () => {
                 <RadioGroup name="game-mode-buttons-group" value={gameMode} onChange={handleGameMode}>
                   <FormControlLabel value="train" control={<Radio />} label={GAME_MODE_LABELS.TRAIN} />
                   <FormControlLabel value="practice" control={<Radio />} label={GAME_MODE_LABELS.PRACTICE} />
-                  <FormControlLabel value="weak" control={<Radio />} label={GAME_MODE_LABELS.WEAK} />
+                  {user.weakCards > 0 && (
+                    <FormControlLabel value="weak" control={<Radio />} label={GAME_MODE_LABELS.WEAK} />
```

Comment:
We should also fix a side effect bug when the weak words are already selected as a gamemode in the appState. If you click play game without selecting a different game mode from the dropdown, the game will still start in weak word mode

Reply:
This shouldn't be a problem. When we navigate back to the homepage, we are setting gamemode to "train" here https://github.com/andreasonny83/jp-flashcards/blob/main/src/Components/Settings.tsx#L35

Alternatively, if we want to be explicit, we could set it here https://github.com/andreasonny83/jp-flashcards/blob/main/src/AppState/useApp.tsx#L176 by retrieving the "new" weakCard list from APi call to Update